### DISCUSSION TITLE
Response is not sent to S3, no error is thrown

### DISCUSSION THREAD

**Discussion by harshil-darji** [aws-powertools/powertools-lambda-java#1005] [Q&A]
I have created an application wherein a lambda is inheriting the AbstractCustomResourceHandler class. It performs certain function on create/update events. I am also generating a presignedUrl and sending it as the ResponseUrl parameter for the CloudFormationCustomResourceEvent. 

However, no output is produced in S3 bucket. For the permissions, I have setup s3 Full access and given it to the lambda function. I also tried overriding the onSendFailure function to perform any logging, but no log output is produced, implying that no error is thrown. Is there any additional S3 configuration I might be missing for this setup to work? Please let me know. Thanks.

**Comment by phipag** [comment]
Hi @harshil-darji, I am currently reviewing some older discussion topics and see that this is >3 years old. Please let me know if this issue still persists. I am resolving the discussion for now.
